.github/ISSUE_TEMPLATE/documentation_report.yml

```diff
@@ -86,18 +86,23 @@ body:
       feature below, *use your best guess if unsure*.
 
 
-      **Tip:** Be advised that the source for some parts of the
-      documentation are hosted outside of this repository. If the page
-      you are reporting describes modules/plugins/etc that are not
-      officially supported by the Ansible Core Engineering team, there
-      is a good chance that it is coming from one of the [Ansible
-      Collections maintained by the community][collections org]. If this
-      is the case, please make sure to file an issue under the
-      appropriate project there instead.
+      **Tip:** Source files for Ansible community documentation is
+      hosted in different repositories. Please make sure to file an
+      issue in the correct project.
+      Documentation for modules/plugins/etc that are officially
+      supported by the Ansible Core Engineering team is available
+      in this (`ansible/ansible`) repository.
+      User documentation pages such as the Playbook Guide,
+      Developer Guide, and so on is available in the
+      [`ansible/ansible-documentation`][ansible-documentation] repository.
+      Documentation for other modules/plugins/etc is likely to be
+      available in one of the [Ansible Collections maintained by
+      the community][collections org].
```

Comment:
Can we change this to say use the 'issue tracker' button on the top collection page to find the repo and open a docs issue there? I believe it's available for all collections:
![image](https://github.com/ansible/ansible/assets/30267855/03ee79d5-8e33-48c7-8cbd-08d7d000869c)

Reply:
Great idea @samccann :tada: